ingestfile

```diff
@@ -26,26 +12,22 @@ _usage(){
     echo "This script will run an interactive interview and then process an input file accordingly."
     echo "Dependencies: ${DEPENDENCIES[@]}"
     echo "Usage: $(basename ${0}) [ -options ]"
-    echo "   default uses a graphical user interface for interactive component of inputing file and media ID, setting volume adjustment, formula, and slate creation."
-    echo "  -e Uses the terminal interface for interactive component of inputing file and Media ID, volume adjustment, formula, and slate creation."
-    echo "  -p Preservation mode- to be used when packaging digitized material and digitization logs. Does not make any deliveries, and creates the package in the same directory as the input."
-    echo "  -n (do not make any deliveries except for sending the final package to ${AIP_STORAGE}.)"
-    echo "  -i PSA Mode- to be used when packaging PSA files. Does not make any deliveries except for sending the final package to ${DELIVERYTOOMNEON}."
+    echo "  Default mode: presents a graphical user interface to specify file, media ID, and other options. Automatically creates broadcast and access copies, and delivers packages to ${OMNEONPATH}, ${PODCASTDELIVER}, ${YOUTUBEDELIVER}, and ${AIP_STORAGE}."
+    echo "  -p Preservation mode: formula to be used when packaging digitized material and digitization logs. Creates access copies, but does not make any deliveries, and creates the package in the same directory as the input."
+    echo "  -i PSA mode: formula to be used when packaging PSA files. Creates access copies, but does not make any deliveries except for sending the final package to ${DELIVERYTOOMNEON}."
+    echo "  -t Terminal mode: user specifies file, media ID, and other options through interactive terminal interface prompts."
     echo "  -h display this help"
     echo
     exit
 }
 
 user_input="${*}"
 OPTIND=1
-
-while getopts ":F:pnhei" OPT ; do
```

Comment:
Here the leading colon was removed, but this indicates "silent error reporting mode". Without it the user would get an error from both getopts and our own `*)` and `:)` cases.

Reply:
I also guess that `:)` has to come before `*)`, otherwise only `*)` will happen.